src/RestEase/RequestPropertyAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace RestEase
+{
+    /// <summary>
+    /// Marks a parameter as HTTP request message property
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter | AttributeTargets.Property, Inherited = false, AllowMultiple = false)]
+    public class RequestPropertyAttribute : Attribute
```

Comment:
I do wonder whether `[RequestProperty]` is a bit confusing as a name. It makes sense in the context of `HttpRequestMessage`, but it's hard to link it to `HttpRequestMessage.Prorties` from its name. Perhaps `HttpRequestMessageProperty` or `RequestMessageProperty`?

Reply:
I had exactly the same thought and you probably noticed that most of the internal members and variables are prefixed with `httpRequestMessage` to give very clean meaning but I thought that for the public attribute the name is a bit long. But since it will be most likely used only very sparsely it's not a problem. I'll rename it.

Reply:
Yeah, I don't have anything against long names in public APIs, so long as they're descriptive (e..g QuerySerializationMethod, RequestQueryParamSerializer, etc)